tests/locations/test_package.py

```diff
@@ -1370,6 +1370,89 @@ def test_move_uncompressed_aip(self):
         assert pkg.current_location == dst_location
 
 
+@pytest.fixture
+@pytest.mark.django_db
+def space(tmp_path):
+    space_dir = tmp_path / "space"
+    space_dir.mkdir()
+
+    staging_dir = tmp_path / "staging"
+    staging_dir.mkdir()
+
+    space = models.Space.objects.create(
+        uuid=uuid.uuid4(),
```

Comment:
Is there a point on initializing the `uuid` attributes in these fixtures? The model fields do it automatically.

Reply:
There is no necessity of doing it. I just did it for no reason. Let me remove it.